### PR TITLE
Added a readme and put this into virtual enviroment 

### PR DESCRIPTION
**Pull request by ghost**
Pls merge

### CODE REVIEW

**Review by mushrom — CHANGES_REQUESTED** [review]
Hmm, as is, I don't feel comfortable merging this. A few issues:

- Commits the `__pycache__` directory, which shouldn't be included
- Includes the virtual environment set up on your machine, when this should be generated on the user's side with the config and a requirements.txt for pip. The reason this is an issue is because the included executables might change, the python symlinks might break even on linux, and it just bloats the repo.
- Smaller sidepoint, the libraries might receive updates which would then need to be manually committed to the repo, when this could automagically be handled by pip.

So, to sum it up, just needs unnecessary directories removed and a requirements.txt added to work with pip.

**Review by mushrom — CHANGES_REQUESTED** [review]
"lightweight" is still misspelled, and should be "a lightweight, Flask-based imageboard system". Sorry to be a stickler but grammer are important :+1:
